rails/empty-arrays-in-json-request.md

```diff
@@ -0,0 +1,66 @@
+Deep Munging
+
+Even when turned off
+
+```ruby
+put "/api/v1/post/1", {
+  post: { tags: [] },
+  format: :json
+}
+```
+
+
+Will remove the entire `post` hash:
+
+```ruby
+{}
+```
+
+Since the `"Content-Type"` is unset, Rack defaults to treat the request as
+`"application/x-www-form-urlencoded"`.
+
+The URL encoded representation of `["foo", "bar"]` is:
+
+```
+?tags[]=foo&tags[]=bar
+```
+
+However, it isn't possible to represent `[]`:
+
+```
+?tags[]=
```

Comment:
Are we sure thats what it would evaluate to? 

Even with deep_munge disabled, the `tags` key ("form" in our case) wasn't being included in params